core/templates/dev/head/pages/about-page/about-page.directive.html

```diff
@@ -483,7 +483,7 @@ <h2 translate="I18N_ABOUT_PAGE_CREDITS_TAB_HEADING"></h2>
                 <li>Zoe Madden-Wood</li>
               </ul>
             </div>
-            <p translate="I18N_ABOUT_PAGE_CREDITS_TAB_TEXT_BOTTOM" translate-values="{listOfNames: '<[$ctrl.listOfNames]>'}">
+            <p translate="I18N_ABOUT_PAGE_CREDITS_TAB_TEXT_BOTTOM" translate-values="{listOfNames: $ctrl.listOfNames}">
```

Comment:
To test:
- Go to /about
- Click the "Credits" tab
- Scroll down and check the end message.
![image](https://user-images.githubusercontent.com/16653571/61579227-41a2c400-ab20-11e9-88be-3521ca1f8e68.png)